### ISSUE TITLE
Results of Chrome Accessibility Audit

### ISSUE THREAD

**Issue by benedfit** [alistapart/pattern-library#7]
If it's of interest I ran the Chrome Accessibility Developer Tools Audit (https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb?hl=en) against the patterns and got the following. Happy to help fix any if wanted
#### Pattern library specific issue

`<select name=​"section" id=​"pattern-select" class=​"nav-section-select">​…​</select>​`
### [Warning] Meaningful images should not be used in element backgrounds (1)

`<label class=​"submit-search" for=​"submit-search">​…​</label>​`
### [Warning] Text elements should have a reasonable contrast ratio (18)

`<a href=​"https:​/​/​etherpad.mozilla.org/​ep/​pad/​view/​ro.gtMA3Rr4lYIam9OU/​latest">​https://etherpad.mozilla.org​</a>​`
#### These are from prism.js

`<span class=​"token selector">​html ​</span>​`
`<span class=​"token punctuation">​{​</span>​`
`<span class=​"token punctuation">​:​</span>`
`<span class=​"token punctuation">​;​</span>​`
`<span class=​"token punctuation">​}​</span>​`
`<span class=​"token selector">​p ​</span>​`
`<span class=​"token punctuation">​{​</span>​`
`<span class=​"token punctuation">​:​</span>​`
`<span class=​"token punctuation">​;​</span>​`
`<span class=​"token punctuation">​}​</span>​`

`<a href=​"#" class=​"embed-link">​Embed​</a>​`
`<h1 class=​"deadly-subtitle">​For people who make websites.​</h1>​`
`<a href=​"/​issue/​386" class=​"issue-number">​…​</a>​`
`<span>​Issue №​</span>​`
`<time class=​"pubdate updated" datetime=​"2014-01-02T08:​00:​37-05:​00">​January 02, 2014​</time>​`
`<a class=​"issue-number" href=​"/​issue/​388">​…​</a>​`
`<span>​Issue №​</span>​`
### [Warning] These elements are focusable but either invisible or obscured by another element (6)

`<a class=​"photo" href=​"/​author/​shorton">​…​</a>​`
`<a class=​"photo" href=​"/​author/​whitneyq">​…​</a>​`
`<a class=​"photo" href=​"/​author/​mikepick">​…​</a>​`
`<a href=​"http:​/​/​www.webydo.com/​?utm_source=deck&utm_medium=cpc&utm_content=faceviking&utm_campaign=a">​…​</a>​`
`<a href=​"http:​/​/​www.thoughtworks.com/​continuous-integration?utm_campaign=snap-beta&utm_medium=cpm&utm_source=the-deck&utm_content=ci-loops">​…​</a>​`
`<a href=​"https:​/​/​www.atlassian.com/​git?utm_source=the-deck&utm_medium=banner&utm_content=learn-the-basics-workflow&utm_campaign=git-microsite">​…​</a>​`

**Comment by benedfit** [comment]
You lot still need a hand with this? Or any of the other issues I raised at the same time?

**Comment by maban** [comment]
I'm going to run through all these shortly – some I've already fixed, just not pulled into this pattern library. I have some time this coming weekend though so might be able to tackle some of these then, and if not, I'll give you a shout.

**Comment by maban** [comment]
Sorry this took so long. The struggle with supporting multiple projects and repos. :(

* [x] Controls and media elements should have labels
* [x] "view this pattern" link is outside of summary, and full text rather than just "#"
* [x] I've tweaked the colours in plugin.js as per @hellogeri's suggestion. There is still room for improvement.

The other issues are with the patterns themselves, and fixing them here alone isn't as good as fixing them on the live site, so I'll check if these are still relevant and create issues where appropriate for the actual site.
